keras_nlp/models/bert/bert_preprocessing.py

```diff
@@ -290,8 +306,36 @@ def from_config(cls, config):
         return cls(**config)
 
     def call(self, x, y=None, sample_weight=None):
-        if not isinstance(x, (list, tuple)):
+        # Check the input type.
```

Comment:
This logic seem tedious to duplicate for each model. I wonder if it could be refactored into a util?

Reply:
Yeah, I was very torn between brevity, and the clarify on having it right on the layer.

But we will need this in a few places, and it will be hard to keep in lock step. Let's go with a utility!